Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
Still wondering… Is `REPRAPWORLD_KEYPAD` an `ULTIPANEL`? My understanding is that it has no display, so is only a `NEWPANEL` with no `ULTIPANEL`. If `ULTIPANEL` is enabled, lots of code pertaining to a non-existent LCD may be compiled in.